data/fwupd.service.in

```diff
@@ -32,4 +32,5 @@ RestrictRealtime=yes
 RestrictSUIDSGID=yes
 SystemCallArchitectures=native
 SystemCallErrorNumber=EPERM
+SystemCallFilter=~@resources
```

Comment:
Won't this mean we have two SystemCallFilter keys?

Reply:
Yes it will - but I did not think this through fully, i.e. how this will interact with the dynamic options. The end desired state will be as below (this is an example I am running for Postfix):

SystemCallFilter=@system-service
SystemCallFilter=~@resources

Mixing and matching whitelist and blacklist is allowed and this operates as expected. What is the best way to handle it here?

Thanks for asking the question and pointing out my mistake.

Reply:
This will work, but will also leave @resources enabled - how would we go about dropping that?

Reply:
I'm confused why @resources would be enabled in the first place. Is @resources part of @system-service?

Reply:
Yes, the system-service group also includes privileged and resources - hence the explicit blacklist of resources.

Reply:
Okay, I think what i'd rather see is `SystemCallFilter=~@resources` also in meson.build too -- the ordering is kinda critical and it'd be too easy to "reformat" the service file and accidentally switch them around.

Reply:
Happy with that, but I am uncertain about the syntax/format here. 

system_call_filter = ['~@resources'] would conflict or override the whitelist line, would it not?

Reply:
Ohh they'll need to be two different lists in meson I think.

Reply:
Can you help me with the syntax to achieve that? They will definitely have to be two different lines at the very least - you cannot combine whitelist and blacklist on one line as far as I understand it. I am also not sure if ordering is important, but either way happy to go with whichever route you think is best.